conanfile.py

```diff
@@ -216,9 +216,9 @@ def requirements(self):
             self.requires("double-conversion/3.1.4@bincrafters/stable")
         if self.options.with_freetype and not self.options.multiconfiguration:
             self.requires("freetype/2.10.0@bincrafters/stable")
-        # if self.options.with_icu:
-        #     self.requires("icu/63.1@bincrafters/stable")
-        #     self.options["icu"].shared = self.options.shared
+        if self.options.with_icu:
+           self.requires("icu/64.2@bincrafters/stable")
```

Comment:
There is a tab/spaces problem with these lines

Reply:
don't see the problem
- no tab in these line, only spaces
- correctly indented under the "if" block

Reply:
That's spaces number problem actually: https://travis-ci.com/bincrafters/conan-qt/jobs/225195746#L2167

Reply:
well spot !! fixed.